static/js/web-install.js

```diff
@@ -148,6 +148,8 @@ const legacyQualcommDevices = ["sunfish", "coral", "flame", "bonito", "sargo", "
 
 const tensorDevices = ["cheetah", "panther", "bluejay", "raven", "oriole"];
 
+const day1SnapshotCancelDevices = ["barbet", "redfin", "bramble", "cheetah", "panther", "bluejay", "raven", "oriole"];
```

Comment:
Can you order this from newest to oldest like the others?

Reply:
done